runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -548,7 +548,9 @@
 
 	SKILLS_CONTAINER(WidgetID.SKILLS_GROUP_ID, WidgetID.Skills.CONTAINER),
 
-	GAUNTLET_TIMER_CONTAINER(WidgetID.GAUNTLET_TIMER_GROUP_ID, WidgetID.GauntletTimer.CONTAINER);
+	GAUNTLET_TIMER_CONTAINER(WidgetID.GAUNTLET_TIMER_GROUP_ID, WidgetID.GauntletTimer.CONTAINER),
+
+	ENCOUNTER_HEALTH_BAR(WidgetID.ENCOUNTER_HEALTH_BAR_GROUP_ID, WidgetID.EncounterHealthBar.CONTAINER);
```

Comment:
This should be named `COX_BOSS_HEALTH_BAR` since it's specific to raids

Reply:
This health bar widget is not unique to raids (Galvek, The Nightmare, Vanstrom Klause). A generic toggle was added in last weeks update so I'd expect Jagex to also use it for other encounters in the future.

![image](https://user-images.githubusercontent.com/66518222/85321069-6045d380-b4c4-11ea-9a8d-aa9687b42faa.png)

![image](https://user-images.githubusercontent.com/66518222/85321731-71431480-b4c5-11ea-9d05-3230ebd0a88d.png)

Reply:
Ah I didn't realize it was generic to all of those. Maybe renaming this to be more in line with Jagex's naming then? `HEALTH_OVERLAY_BAR`?